pupepat/fitting.py

```diff
@@ -109,12 +122,12 @@ def fit_cutout(data, source, plot_filename, image_filename, header, id, fit_circ
     x0 = y0 = cutout_radius + 1.0
     r = cutout_coordinates(cutout, x0, y0)
 
-    # Run sep again and make sure there is only one source in the cutout
-    cutout_sources = run_sep(cutout, mask_threshold=25.0 * np.sqrt(background) + background)
+    # repair any hot pixels and/or cosmic rays from the cutout
+    # TODO: evaluate if this could be done even more beneficial earlier in the pipeline on the whole image...)
+    hpcr_mask, cutout = astroscrappy.detect_cosmics(cutout, readnoise=10., gain=1.0, sigclip=6, sigfrac=1.0)
```

Comment:
sigfrac might be ok at 0.5. Readnoise should come from the header.

Reply:
done.